scripts/sync-workflow.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
I think we can't be sure of the naming at the moment because it would depend on how we're going to split the actions each script does...

Along with cache invalidation this is the hardest thing to do, after all.